Example_Programs/programming_concpets/calculator_complex/test/test_complex_calculator.c

```diff
@@ -61,6 +99,18 @@ void test_negative(void)
     TEST_ASSERT_EQUAL(-100, result.real);
     TEST_ASSERT_EQUAL(-110, result.imaginary);
 
+    TEST_ASSERT_EQUAL(SUCCESS, complex_diff(&c1, &c2, &result));
+    TEST_ASSERT_EQUAL(50, result.real);
+    TEST_ASSERT_EQUAL(80, result.imaginary);
+    
+    TEST_ASSERT_EQUAL(SUCCESS, complex_mul(&c1, &c2, &result));
+    TEST_ASSERT_EQUAL(450, result.real);
+    TEST_ASSERT_EQUAL(3500, result.imaginary);
+    
+    TEST_ASSERT_EQUAL(SUCCESS, complex_div(&c1, &c2, &result));

```

Comment:
Consider testing with more values. Ex: all values are 0, or only imaginary part is 0. All corner cases.

Reply:
okay sir. I will write more test cases for each operations.